server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/NodeType.java

```diff
@@ -52,5 +52,7 @@
   SUBROUTINE,
   SUBROUTINE_NAME_NODE,
   PARAGRAPH_NAME_NODE,
-  SECTION_NAME_NODE;
+  SECTION_NAME_NODE,
+  REMARKS,
```

Comment:
There is no need to define a special type for the remarks. OBSOLETE will be enough for the entire set of these nodes.